dev/tools/gen_date_localizations.dart

```diff
@@ -0,0 +1,147 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
There should be a comment about what this app does, when it needs to be run, how to run it.